docs/user_manual/boxes.md

```diff
@@ -0,0 +1,328 @@
+
+Select the target position in the Contents diagram. In the information section that appears adjacent to the
+Contents diagram (see figure {{figure}}), enter the item's name, alias, or barcode into the "Search" field and
+click "Lookup". 
+
+{% include userman-figure.md num=figure cap="Additional information when one position is selected"
+img="one-position-selected.png" %}
+
+Once the search completes, any matching items found will be in the "Results" dropdown. Select the target item
+from the dropdown and click "Update Position". After the update completes, the information about the item in
+the selected position will update with the target item's information.
+
+If the target position already contains an item, saving the new item to that position will cause the existing
+item's location to become unknown. If the existing item's location should be tracked, it should be moved to a
+new box before or after saving the new item to the target position.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% assign figure = figure | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Adding Multiple Items to a Box" %}
+
+To add multiple items to a box, go to the Edit Box page by clicking on the box name in the Boxes list page.
+Select the target positions in the Contents diagram. Find the Position and Search rows adjacent to the
+Contents diagram (see figure {{figure}}), and enter the target item's name, alias, or barcode into the Search
+field for the target position. 
+
+{% include userman-figure.md num=figure cap="Additional information when multiple positions are selected"
+img="multiple-positions-selected.png" %}
+
+Once all Search fields are filled out, click "Update" below the last search field. If the target
+position already contains an item, a dialog box will name the items that already in those positions and ask
+you to confirm that you wish to replace them with the items matching your search terms. If any items to be
+replaced are not being re-added to a different position in the box, they will be removed from the box. Click
+"Replace" to continue, or "Cancel" to cancel the update and select new positions.
+
+If one item is found for each position, the page is refreshed and the Contents diagram and table will contain
+the target items. If more than one item is found for a given position, or if no matching item can be found for
+a given search term, no items will be updated and a dialog box will display the search terms which were in
+error.
+
+Multiple items can also be added to a box using the [Scanning a box](#scanning_a_box) instructions.
+
+{% include userman-toplink.md %}
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Removing a Single Item from a Box" %}
+
+A single item can be removed from a box, which removes its Location value. It is advised to scan it into
```

Comment:
On the Edit Sample page, the "Location" field is `locationBarcode` and isn't affected by removing the sample from a box